codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/generators/config/StalledStreamProtectionConfigCustomization.kt

```diff
@@ -124,7 +124,7 @@ class StalledStreamProtectionOperationCustomization(
                         // we can't count on hyper to poll a request body on wake.
                         rustTemplate(
                             """
-                            #{StalledStreamProtectionInterceptor}::new(#{Kind}::ResponseBody)
```

Comment:
do you think we need a BMV to enable it on request bodies?

Reply:
Yeah, I'm not sure. I guess it depends on whether or not we consider this a bug fix?

Reply:
Filed https://github.com/smithy-lang/smithy-rs/issues/3510 to track the BMV change.